src/pages/profile.tsx

```diff
@@ -0,0 +1,12 @@
+import useAuthentication from 'hooks/use-authentication'
+import BaseNewLayout from 'layouts/base-new'
+import ProfileView from 'views/profile-view'
+
+const ProfilePage = () => {
+	const { user } = useAuthentication()
+
+	return <ProfileView user={user} />
+}
```

Comment:
Question: Curious why you're passing `user` down here as opposed to using the hook in the nested `ProfileView`?

Reply:
I did it to make the view component a little more agnostic. No strong feelings either way; would you like me to update the view to use the hook instead?

Reply:
No action needed, this is fine for now!